notes.sql

```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE notes_db;
+
+CREATE TABLE notes_tb {
+    title varchar(100) NOT NULL,
+    note varchar(200) NOT NULL,
```

Comment:
200 is to small for the note

Reply:
Haha it might be! Have modified it to 1000. Thanks!